testData/preferences/explodeLaunchHandlerStart.json5

```diff
@@ -5,6 +5,7 @@
             "gpii-default": {
                 "name": "Default preferences",
                 "preferences": {
+                    "http://registry.gpii.net/applications/net.gpii.explode/enabled": true,
```

Comment:
Does this have to be expressed as a full URL rather than as a property under `net.gpii.explode`?

Reply:
Looks like it does.  I wish it didn't, it means all of the /enabled stuff is much more difficult to validate.  It also means we have no good way of indicating which things can be enabled at all, i.e. we can pass along any solution's URL with /enabled at the end whether or not it's meaningful.

Reply:
OK, we could investigate how much rework would be required to bundle this in as a standard setting. As I recall, we didn't want to pollute the existing namespace of settings in case that name was used by something else - but I guess transforms are sufficiently mature that these could easily be diverted. Historically, I think that "/enabled" was an initial attempt to clean up what had been pretty irregular URLs under the old "common terms" system such as /magnifierEnabled

Reply:
If it's too much work to make this work like any other setting we should at least discuss whether/how we can clearly indicate which things can be /enabled and add detection.  i.e. if what happens if someone tries to enable something that is always running and which cannot be stopped?  What if they pass that for a solution that has no launch handlers at all?